extensions/spark/kyuubi-spark-authz/src/test/scala/org/apache/kyuubi/plugin/spark/authz/ranger/RangerSparkExtensionSuite.scala

```diff
@@ -996,4 +997,46 @@ class HiveCatalogRangerSparkExtensionSuite extends RangerSparkExtensionSuite {
       }
     }
   }
+
+  test("InsertIntoHiveDirCommand") {
+    val db1 = defaultDb
+    val table1 = "table1"
+    val tableDirectory = getClass.getResource("/").getPath + "table_directory"
+    val directory = File(tableDirectory).createDirectory()
```

Comment:
when will this be cleaned？

Reply:
> when will this be cleaned？

Just follow other ut...seems not cleaned

Reply:
Let's have a function to do something like `withTempDir`

Reply:
Updated

Reply:
> Let's have a function to do something like `withTempDir`

Done